features/stamp_duty.feature

```diff
@@ -12,51 +12,139 @@ Scenario: Welsh users
   Given I visit the Welsh Stamp Duty page
   Then  I see the Welsh stamp duty calculator
 
-Scenario: House price which is less than £125,000
+Scenario: House price which is less than £40,000 for first home
```

Comment:
Good shout. We're just about to implement them on this new feature we're working on, so will double back to look at them here after that.